### PR TITLE
[release]1.15.0 string fetch

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Fetch strings for 1.15.0
# Why

updates any old strings

https://skype.visualstudio.com/SPOOL/_workitems/edit/3664500
# How Tested

### CODE REVIEW

File: packages/react-composites/src/composites/localization/locales/ar-SA/strings.json [hunk 1]
```diff
@@ -324,11 +324,18 @@
...
+    "leaveConfirmDialogTitle": "هل تريد مغادرة المكالمة؟",
+    "leaveConfirmDialogContent": "هل تريد بالتأكيد مغادرة المكالمة؟",
+    "endCallConfirmDialogTitle": "هل تريد إنهاء المكالمة؟",
+    "endCallConfirmDialogContent": "ستنهي المكالمة لدى الجميع.",
+    "invalidMeetingIdentifier": "معرف اجتماع أو رمز مرور غير صالح"
   },
   "chat": {
     "chatListHeader": "في هذه الدردشة",
-    "uploadAttachment": "تحميل ملف"
+    "uploadAttachment": "Upload Attachment"
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/localization/locales/ar-SA/strings.json:338]: Missing translation here

File: packages/react-composites/src/composites/localization/locales/cs-CZ/strings.json [hunk 2]
```diff
@@ -324,11 +324,18 @@
...
+    "leaveConfirmDialogTitle": "Chcete z hovoru odejít?",
+    "leaveConfirmDialogContent": "Opravdu chcete hovor opustit?",
+    "endCallConfirmDialogTitle": "Chcete hovor ukončit?",
+    "endCallConfirmDialogContent": "Hovor ukončíte pro všechny.",
+    "invalidMeetingIdentifier": "Neplatné ID schůzky nebo heslo"
   },
   "chat": {
     "chatListHeader": "V tomto chatu",
-    "uploadAttachment": "Nahrát soubor"
+    "uploadAttachment": "Upload Attachment"
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/localization/locales/cs-CZ/strings.json:338]: missing translation here

File: packages/react-composites/src/composites/localization/locales/es-ES/strings.json [hunk 3]
```diff
@@ -324,11 +324,18 @@
...
+    "leaveConfirmDialogTitle": "¿Abandonar la llamada?",
+    "leaveConfirmDialogContent": "¿Está seguro de que desea abandonar la llamada?",
+    "endCallConfirmDialogTitle": "¿Finalizar la llamada?",
+    "endCallConfirmDialogContent": "Finalizará la llamada para todos los usuarios.",
+    "invalidMeetingIdentifier": "Id. de reunión o código de acceso no válidos"
   },
   "chat": {
     "chatListHeader": "En este chat",
-    "uploadAttachment": "Subir archivo"
+    "uploadAttachment": "Upload Attachment"
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/localization/locales/es-ES/strings.json:338]: Okay, not gonna comment about this one anymore cause looks to be missing on a lot of them at the moment. Might need to wait a bit more until this gets translated

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/localization/locales/es-ES/strings.json]: image
looks like refetching fixed the issue

File: packages/react-composites/src/composites/localization/locales/pt-BR/strings.json [hunk 4]
```diff
@@ -324,11 +324,18 @@
...
+    "leaveConfirmDialogTitle": "Sair da chamada?",
+    "leaveConfirmDialogContent": "Tem certeza de que deseja sair da chamada?",
+    "endCallConfirmDialogTitle": "Encerrar a chamada?",
+    "endCallConfirmDialogContent": "Você encerrará a chamada para todos.",
+    "invalidMeetingIdentifier": "ID ou senha de reunião inválida"
   },
   "chat": {
     "chatListHeader": "Neste chat",
-    "uploadAttachment": "Carregar Arquivo"
+    "uploadAttachment": "Upload do Anexo"
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/localization/locales/pt-BR/strings.json:338]: Looks like a regression here

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**